python/common/common/time.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) ZeroC, Inc.
+
+"""
+Utility functions to convert between datetime objects and custom timestamps
```

Comment:
One difficulty here is Python uses the term timestamp already, with a different meaning:
https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp

And, as far as can tell, it doesn't use the term time point.

Then, it appears that the Jan 1, 0001 model we use - the proleptic Gregorian calendar - is actually the default for Python. So we probably don't need these functions at all.

Reply:
I guess something like:

```
(datetime.now() - datetime.combine(date.fromordinal(1), datetime.min.time())).total_seconds()
```

And then convert seconds to ticks.

Reply:
I would update the description. Something like:
```
Utility functions to convert between datetime and custom timestamps based on the Proleptic Gregorian calendar.
```